### DISCUSSION TITLE
First render using MiEx in Maya/Renderman!

### DISCUSSION THREAD

**Discussion by barelyaiden** [BramStoutProductions/MiEx#99] [Show and tell]
This was pretty fun to work on!! It took a couple of head scratches to figure everything out but here's the final result!
I only ever changed the water material from the default export and I used compositing to add the fog in the distance. The clouds I ended up just exporting from jmc2obj and that's about it!

![RenderWorld1](https://github.com/user-attachments/assets/25ccef99-2c48-4d99-8381-639d0bc65551)

**Comment by TheiOpTicX** [comment]
i cant open the jar file, how do i open it

**Reply by PurpBatBoi** [reply]
This is not the place to ask this. You need **Java** in order to run this software.
